l1-contracts/test/Decoder.t.sol

```diff
@@ -167,22 +166,14 @@ contract DecoderTest is Test {
       hex"0000002400000008aafdc7aa93e78a700000001497aee30906a86173c86c6d3f108eefc36e7fb014";
     (bytes32 logsHash, uint256 bytesAdvanced) = helper.computeKernelLogsHash(emptyKernelData);
 
-    bytes32 referenceLogsHashFromIteration1 = bytes32(
-      uint256(
-        sha256(
-          hex"0000000000000000000000000000000000000000000000000000000000000000aafdc7aa93e78a70"
-        )
-      ) % Constants.P
+    bytes32 referenceLogsHashFromIteration1 = Hash.sha256ToField(
+      abi.encodePacked(
+        hex"0000000000000000000000000000000000000000000000000000000000000000aafdc7aa93e78a70"
```

Comment:
Should the raw bytes from here be a variable somewhere instead? We are using it in both here and in line 150.